### PR TITLE
Added LogInTest method

### PR DESCRIPTION
**Pull request by VAdrianStan**
- Added new LogInTest method
- added new constant validUsername
-

### CODE REVIEW

File: src/main/java/page/objects/LogInPage.java [hunk 1]
```diff
@@ -12,4 +13,15 @@ public void clickCreateAccountButton() {
...
+        setText(By.cssSelector("#email"), Constants.VALID_UESR_NAME);
+    }
+
+    public void inputPassword() {
+        setText(By.cssSelector("#pass"), Constants.PASSWORD);
+    }
+
+    public void clickSignInButton() {
+        click(By.cssSelector(".fieldset.login .actions-toolbar .primary .action.login.primary"));
+    }
```
Comment (GabrielCorlan) [inline src/main/java/page/objects/LogInPage.java:26]: CSS locator is too long. Please find a way to make it shorter

File: src/main/java/page/objects/SignInPage.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
+package page.objects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class SignInPage extends BasePage{
+
+
+    public SignInPage(WebDriver driver) {
```
Comment (GabrielCorlan) [inline src/main/java/page/objects/SignInPage.java:7]: Remove extra spaces

File: src/main/java/page/objects/SignInPage.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        super(driver);
+    }
+
+    public void clickSignInButton(){
+        click(By.cssSelector(".panel.wrapper .authorization-link "));
+    }
+
+
+
+}
```
Comment (GabrielCorlan) [inline src/main/java/page/objects/SignInPage.java:13]: Same as above

File: src/test/java/BaseTest.java [hunk 4]
```diff
@@ -28,13 +28,14 @@ public void setUp() throws Exception {
         logInPage = new LogInPage(driver);
         createAccountPage = new CreateAccountPage(driver);
         myAccountPage = new MyAccountPage(driver);
+        signInPage = new SignInPage(driver);
 
 
     }
```
Comment (GabrielCorlan) [inline src/test/java/BaseTest.java:34]: Same as above

**Review by GabrielCorlan — CHANGES_REQUESTED** [review]
Please review the comments

File: src/main/java/page/objects/LogInPage.java (hunk 1 above)
Comment (VAdrianStan) [reply src/main/java/page/objects/LogInPage.java:26]: done

File: src/main/java/page/objects/SignInPage.java (hunk 2 above)
Comment (VAdrianStan) [reply src/main/java/page/objects/SignInPage.java:7]: done

File: src/main/java/page/objects/SignInPage.java (hunk 3 above)
Comment (VAdrianStan) [reply src/main/java/page/objects/SignInPage.java:13]: done

File: src/test/java/BaseTest.java (hunk 4 above)
Comment (VAdrianStan) [reply src/test/java/BaseTest.java:34]: done

**Review by GabrielCorlan — APPROVED** [review]
Well done! You can merge into master now. Approved!

**Pull request merged**